src/themes/modern/design-tokens.yaml

```diff
@@ -202,3 +203,6 @@ elevation:
   "24":
     shadow-color: *shadow-color
     shadow-size: *shadow-size-24
+
+box:
+  border-radius: *border-radius-md
```

Comment:
i think we would gain some flexibility here if we added a level to make it "box-border-radius-default", it's possible we would want to add a smaller radius e.g. for mobile

Reply:
Fixed.